src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
+        return Colors.values()[index].toString();
```

Comment:
It’s better to use a standard method of enum `name()` that will be returning String representation of the concrete enum constant.
```suggestion
        return colors[index].name();
```